doc/source/ray-core/scheduling/placement-group.rst

```diff
@@ -92,22 +93,15 @@ Ray placement group can be created via the ``ray.util.placement_group`` (Python)
 
       ray::PlacementGroup pg = ray::CreatePlacementGroup(options);
 
-.. important:: Each bundle must be able to fit on a single node on the Ray cluster.
-
-Placement groups are atomically created - meaning that if there exists a bundle that cannot fit in any of the current nodes, then the entire placement group will not be ready.
+You can block your program until the placement group is ready using the :func:`ready <ray.util.placement_group.PlacementGroup.ready>`.  (compatible with ``ray.get``) or :func:`ready <ray.util.placement_group.PlacementGroup.wait>` (block the program until the placement group is ready) API. 
```

Comment:
```suggestion
You can block your program until the placement group is ready using :func:`ready <ray.util.placement_group.PlacementGroup.ready>` (compatible with ``ray.get``), or :func:`ready <ray.util.placement_group.PlacementGroup.wait>` (which blocks the program until the placement group is ready) API. 
```

Reply:
Actually it's ready and wait, not ready and ready.